packages/addons/src/plugins/localStoragePlugin.ts

```diff
@@ -94,6 +96,7 @@ export function createLocalStoragePlugin(
       const maxAge = localStorageOptions?.maxAge ?? 3600000 // 1 hour
       const key = localStorageOptions?.key ? `-${localStorageOptions.key}` : '' // for scoping to a specific user
       const storageKey = `${prefix}${key}-${node.name}`
+      const clearOnSubmit = localStorageOptions?.clearOnSubmit ?? true
```

Comment:
@GerryWilko Wouldn't this always compute to `true` because if a user set the option to `false` this check would fail and fall back to `true`? 

We can use FormKit's `undefine()` utility for this.
https://formkit.com/api-reference/formkit-utils#undefine

```suggestion
      const clearOnSubmit = undefine(localStorageOptions?.clearOnSubmit)
```

Reply:
Nevermind — it's too early for me to be doing code reviews, haha 😅